### PR TITLE
fix: updated translations for it-IT

### PR DESCRIPTION
**Pull request by bugalot**
This PR aims at implementing [LSI-1770](https://oat-sa.atlassian.net/browse/LSI-1770) and [LSI-1771](https://oat-sa.atlassian.net/browse/LSI-1771).

[LSI-1770]: https://oat-sa.atlassian.net/browse/LSI-1770?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LSI-1771]: https://oat-sa.atlassian.net/browse/LSI-1771?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by SergiiTao — APPROVED** [review]
LGTM

File: locales/it-IT/messages.po [hunk 1]
```diff
@@ -391,3 +391,5 @@ msgstr "Il tuo sistema non è ottimale, si prega di contattare il proprio ammini
 msgid "Your system requires a compatibility update, please contact your system administrator."
 msgstr "Il tuo sistema richiede un aggiornamento, si prega di contattare il tuo amministratore di sistema"
 
+msgid "This browser is supported."
+msgstr "Questo browser è supportato"
```
Comment (mianchenia) [inline locales/it-IT/messages.po:395]: Is `.` missed?

**Review by mianchenia — APPROVED** [review]

File: locales/it-IT/messages.po (hunk 1 above)
Comment (bugalot) [reply locales/it-IT/messages.po:395]: Yes, but it's actually wanted, otherwise it will not match the rest of the GUI

Comment (taorafael) [inline locales/it-IT/messages.po:395]: In Portuguese, "browser" is called "navegador", I've been looking at examples of "navigatore" for Italian. Which one do we keep?

**Pull request merged**